### PR TITLE
Working

### PR DESCRIPTION
**Pull request by TrevorJamesH**
Build the front end of our Simon game

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
+.target{
+    height:150px;
+    width:150px;
+}
+
+#Start{
+    height:50px;
```
Comment (nicosesma) [inline style.css:7]: Is this indented too far?

File: scripts [hunk 2]
```diff
@@ -0,0 +1 @@
+<script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```
Comment (nicosesma) [inline scripts:1]: do you need this file?

File: script.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      $(this).addClass("light");
+      unClickableAll();
+      console.log('clicked');
+
+      setTimeout(function () {
+        $(".target").removeClass("light");
+        clickableAll();
+      }, 3000);
+
+  }
```
Comment (jason00111) [inline script.js:27]: The tabbing is off on lines 8 through 17, makes it a little harder to read.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  });
+
+  var unClickableAll = function(){
+    console.log('unclickable');
```
Comment (nicosesma) [inline script.js]: you can remove the ```console.log()``` I'm glad it worked though

File: scripts (hunk 2 above)
Comment (jason00111) [inline scripts:1]: I recommend either `src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"` or `src="cdnjs.cloudflare.com/ajax/libs/jquery/2.1.3/jquery.min.js"`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+  </div>
+  <div class="buttons" id="buttons">
+    <div class="row">
+      <button class="btn btn-default clickable target green" id="Green">Green</button>
+      <button class="btn btn-default clickable target red" id="Red">Red</button>
+    </div>
+    <div class="row">
+      <button class="btn btn-default clickable target yellow" id="Yellow">Yellow</button>
+      <button class="btn btn-default clickable target blue" id="Blue">Blue</button>
```
Comment (jason00111) [inline index.html]: The classes `btn` and `btn-default` don't seem to be being used.  Do you plan to use them in the future?

**Review by jason00111 — COMMENTED** [review]
Looks like a good start!  I made some very minor suggestions.

File: script.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+      $(this).addClass("light");
+      unClickableAll();
+      console.log('clicked');
+
+      setTimeout(function () {
+        $(".target").removeClass("light");
+        clickableAll();
+      }, 3000);
+
```
Comment (nicosesma) [inline script.js:10]: This is great, specially the fact that the responsibility for the functions has been separated

**Pull request merged**